packages/frontend/src/ee/features/embed/SettingsEmbed/EmbedDashboardsAndChartsForm.tsx

```diff
@@ -1,31 +1,35 @@
 import {
     type DashboardBasicDetails,
     type DecodedEmbed,
+    type SavedChart,
     type UpdateEmbed,
 } from '@lightdash/common';
 import { Button, Flex, MultiSelect, Stack, Switch } from '@mantine/core';
 import { useForm } from '@mantine/form';
 import React, { type FC } from 'react';
 
-const EmbedDashboardsForm: FC<{
+const EmbedDashboardsAndChartsForm: FC<{
```

Comment:
Could this simply be called EmbedContentForm?

Reply:
Oh yes, I renamed it here (by accident) https://github.com/lightdash/lightdash/pull/17940/files#diff-e14ac16d07ee89ead7cdd61502f5a8d75bece4a8a90f6cfc195fa97979ce4a84